a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -1019,31 +1019,358 @@ <h4>Display statements</h4>
+									
+									<span data-localization-id="conformance-level-aa"
+										data-localization-mode="compact"> Level AA</span>
+								</p>
+								<p data-localization-mode="descriptive">
+									<span data-localization-id="conformance-claim"
+										data-localization-mode="descriptive">This
+										publication claims to meet</span>
+									
+									<span data-localization-id="conformance-epub-accessibility-1-1"
+										data-localization-mode="descriptive"> EPUB Accessibility 1.1</span>
+									
+									<span data-localization-id="conformance-wcag-2-0"
+										data-localization-mode="descriptive"> Web Content Accessibility Guidelines (WCAG) 2.0</span>
+									
+									<span data-localization-id="conformance-level-aa"
+										data-localization-mode="descriptive"> Level AA</span>
+								</p>
+							</dd>
+							<dt>EPUB Accessibility 1.1 WCAG 2.0 Level AAA</dt>
+							<dd>
+								<p data-localization-mode="compact">
+									<span data-localization-id="conformance-claim"
+										data-localization-mode="compact">This
+										publication claims to meet</span>
+									
+									<span data-localization-id="conformance-epub-accessibility-1-1"
+										data-localization-mode="compact"> EPUB Accessibility 1.1</span>
+									
+									<span data-localization-id="conformance-wcag-2-0"
+										data-localization-mode="compact"> WCAG 2.0</span>
+									
+									<span data-localization-id="conformance-level-aaa"
+										data-localization-mode="compact"> Level AAA</span>
+								</p>
+								<p data-localization-mode="descriptive">
+									<span data-localization-id="conformance-claim"
+										data-localization-mode="descriptive">This
+										publication claims to meet</span>
+									
+									<span data-localization-id="conformance-epub-accessibility-1-1"
+										data-localization-mode="descriptive"> EPUB Accessibility 1.1</span>
+									
+									<span data-localization-id="conformance-wcag-2-0"
+										data-localization-mode="descriptive"> Web Content Accessibility Guidelines (WCAG) 2.0</span>
+									
+									<span data-localization-id="conformance-level-aaa"
+										data-localization-mode="descriptive"> Level AAA</span>
+								</p>
+							</dd>
+							<dt>EPUB Accessibility 1.1 WCAG 2.1 Level A</dt>
+							<dd>
+								<p data-localization-mode="compact">
+									<span data-localization-id="conformance-claim"
+										data-localization-mode="compact">This
+										publication claims to meet</span>
+									
+									<span data-localization-id="conformance-epub-accessibility-1-1"
+										data-localization-mode="compact"> EPUB Accessibility 1.1</span>
+									
+									<span data-localization-id="conformance-wcag-2-1"
+										data-localization-mode="compact"> WCAG 2.1</span>
+									
+									<span data-localization-id="conformance-level-aa"
+										data-localization-mode="compact"> Level AA</span>
```

Comment:
This is in the A section, and the AA section is missing.

Reply:
Ya, not sure what happened there. Looks like I might have accidentally highlighted and deleted the text while fixing up the IDs. Should be fixed now.